src/tokenizer/tokenizer.py

```diff
@@ -795,14 +803,19 @@ def parse_tokens(txt, **options):
     # 7) The process is repeated from step 4) until the current raw token is
     #    exhausted. At that point, we obtain the next token and start from 2).
 
-    for w in gen(txt, replace_composite_glyphs, replace_html_escapes):
+    for w in gen(txt, replace_composite_glyphs, replace_html_escapes, onesentperline):
 
         # Handle each sequence w of non-whitespace characters
 
         if not w:
             # An empty string signals an empty line, which splits sentences
             yield TOK.Split_Sentence()
             continue
+        if w == " ":
```

Comment:
This may not be needed cf. above comments

Reply:
Wasn't needed, deleted!